specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-08-01/examples/WorkbookManagedAdd.json

```diff
@@ -72,21 +62,17 @@
           "userAssignedIdentities": {
             "/subscriptions/6b643656-33eb-422f-aee8-3ac145d124af/resourceGroups/eu2cgroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/id1": {
```

Comment:
are these real resource id/etc values?  or is this all fake?  should all these guids be replaced with fake/0 guid?